tools/tm-bench/transacter.go

```diff
@@ -194,7 +194,7 @@ func (t *transacter) sendLoop(connIndex int) {
 				c.SetWriteDeadline(now.Add(sendTimeout))
 				err = c.WriteJSON(rpctypes.RPCRequest{
 					JSONRPC: "2.0",
-					ID:      rpctypes.JSONRPCStringID("tm-bench"),
+					ID:      rpctypes.JSONRPCIntID((connIndex * 10000) + i),
```

Comment:
Given the current system, 10k here seems to allow for good "padding" around tx IDs, but one could just as easily use `t.Rate` instead of 10k, right?